### ISSUE TITLE
Python bindings fails to test

### ISSUE THREAD

**Issue by mkunz7** [sandia-minimega/minimega#492]
```
root@blue12:~/minimega-master# ./all.bash 
BUILD PACKAGES (linux)
apigen
gonetflow
gopacket
gopcap
gopnm
goreadline
igor
iomeshage
ipmac
meshage
miniccc
minicli
miniclient
minidoc
minilog
minimega
minipager
minitest
minitunnel
nbd
powerbot
present
protonuke
qmp
ranger
ranges
resize
rfbplay
ron
ssh
telnet
version
vlans
vmbetter
vmconfig
vnc
vncdrone
websocket

BUILD PACKAGES (windows)
protonuke
miniccc

BUILD PYTHON BINDINGS
minimega.py - OK

TESTING
?   	apigen	[no test files]
?   	gonetflow	[no test files]
ok  	gopacket	0.005s
?   	gopcap	[no test files]
ok  	gopnm	0.036s
?   	goreadline	[no test files]
?   	igor	[no test files]
?   	iomeshage	[no test files]
?   	ipmac	[no test files]
?   	meshage	[no test files]
?   	miniccc	[no test files]
ok  	minicli	0.008s
?   	miniclient	[no test files]
?   	minidoc	[no test files]
ok  	minilog	1.006s
ok  	minimega	0.028s
?   	minipager	[no test files]
?   	minitest	[no test files]
ok  	minitunnel	0.018s
?   	nbd	[no test files]
?   	powerbot	[no test files]
ok  	present	0.008s
?   	protonuke	[no test files]
?   	qmp	[no test files]
?   	ranger	[no test files]
ok  	ranges	0.005s
ok  	resize	0.128s
ok  	rfbplay	0.010s
?   	ron	[no test files]
ok  	ssh	0.968s
?   	telnet	[no test files]
?   	version	[no test files]
ok  	vlans	0.040s
?   	vmbetter	[no test files]
ok  	vmconfig	0.007s
ok  	vnc	0.008s
?   	vncdrone	[no test files]
ok  	websocket	0.032s
FAIL	minimega.py

BUILD DOCUMENTATION
apigen
```

With debugging enabled in the test file

```
root@blue12:~/minimega-master/misc/python# ./test_minimega.py 
generating python API file...
starting minimega...
EEEEEEEkilling minimega

======================================================================
ERROR: test_listArgs (__main__.TestMinimega)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./test_minimega.py", line 61, in setUp
    except FileNotFoundError:
NameError: global name 'FileNotFoundError' is not defined

======================================================================
ERROR: test_missingArgs (__main__.TestMinimega)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./test_minimega.py", line 61, in setUp
    except FileNotFoundError:
NameError: global name 'FileNotFoundError' is not defined

======================================================================
ERROR: test_noArgs (__main__.TestMinimega)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./test_minimega.py", line 61, in setUp
    except FileNotFoundError:
NameError: global name 'FileNotFoundError' is not defined

======================================================================
ERROR: test_optionArgs (__main__.TestMinimega)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./test_minimega.py", line 61, in setUp
    except FileNotFoundError:
NameError: global name 'FileNotFoundError' is not defined

======================================================================
ERROR: test_optionalArgs (__main__.TestMinimega)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./test_minimega.py", line 61, in setUp
    except FileNotFoundError:
NameError: global name 'FileNotFoundError' is not defined

======================================================================
ERROR: test_streaming (__main__.TestMinimega)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./test_minimega.py", line 61, in setUp
    except FileNotFoundError:
NameError: global name 'FileNotFoundError' is not defined

======================================================================
ERROR: test_stringArgs (__main__.TestMinimega)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./test_minimega.py", line 61, in setUp
    except FileNotFoundError:
NameError: global name 'FileNotFoundError' is not defined

----------------------------------------------------------------------
Ran 7 tests in 1.281s

FAILED (errors=7)
```

**Comment by x64x6a** [comment]
I can't seem to recreate this issue:
```
root@ubuntu:~/minimega/misc/python# ./test_minimega.py 
generating python API file...
starting minimega...
.......killing minimega

----------------------------------------------------------------------
Ran 7 tests in 1.201s

OK
```

**Comment by jcrussell** [comment]
Looks like a Python2 vs Python3 issue.

**Comment by x64x6a** [comment]
I just ran tests with python3 and it seemed to work for me still.

Looks like it can't find minimega?

[https://github.com/sandia-minimega/minimega/blob/master/misc/python/test_minimega.py#L61](https://github.com/sandia-minimega/minimega/blob/master/misc/python/test_minimega.py#L61)

**Comment by x64x6a** [comment]
http://stackoverflow.com/questions/21367320/searching-for-equivalent-of-filenotfounderror-in-python-2

Since 2 doesn't have FileNotFound, would using ```IOError``` instead fix the issue?

**Comment by djfritz** [comment]
Will be resolved by #328 - closing
